airflow/jobs/scheduler_job.py

```diff
@@ -1759,3 +1759,46 @@ def _emit_pool_metrics(self, session: Session = None) -> None:
     @provide_session
     def heartbeat_callback(self, session: Session = None) -> None:
         Stats.incr('scheduler_heartbeat', 1, 1)
+
+    @provide_session
+    def reset_state_for_orphaned_tasks(self, session: Session = None):
+        """
+        Reset any TaskInstance still in QUEUED state that were enqueued by a
+        SchedulerJob that is no longer running.
+        """
+        timeout = conf.getint('scheduler', 'scheduler_health_check_threshold')
```

Comment:
Can we add this entry to config.yml

Reply:
or do we want to have a fallback value here

Reply:
I thought this already existed - I'll have to check

Reply:
oh you are right, sorry